openstudiocore/src/utilities/core/ApplicationPathHelpers.cxx.in

```diff
@@ -146,7 +146,20 @@ namespace openstudio {
     #else
       Dl_info info;
       if (dladdr("GetCurrentModule", &info)) {
-          openstudioDirPath = completeAndNormalize(toPath(info.dli_fname));
+          auto path = toPath(info.dli_fname);
+          if ( path.parent_path().empty() ) {
+            std::istringstream pathstream( getenv("PATH") );
+            std::string pathstring;
+            while ( std::getline(pathstream, pathstring, ':') ) {
+              auto maybepath = toPath(pathstring) / path;
+              if( openstudio::filesystem::exists( maybepath ) ) {
+                openstudioDirPath = maybepath;
```

Comment:
Can elements in PATH be relative?  Might need to complete and normalize this?

Reply:
I don't think they can be relative. but they can probably be links.